toolchain/check/import_cpp.cpp

```diff
@@ -179,7 +179,8 @@ class CarbonClangDiagnosticConsumer : public clang::DiagnosticConsumer {
 // TODO: Consider to always have a (non-null) AST.
 static auto GenerateAst(Context& context, llvm::StringRef importing_file_path,
                         llvm::ArrayRef<Parse::Tree::PackagingNames> imports,
-                        llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> fs)
+                        llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> fs,
```

Comment:
This seems OK for now, but I'm worried that we're going to end up plumbing a lot of clang configuration information through check. Let's talk in the interop meeting about options here.

Reply:
As discussed in the meeting, we're leaving this as is for now and reevaluate when we use pcms (modules).